framework/source/class/qx/ui/window/Window.js

```diff
@@ -618,12 +732,15 @@ qx.Class.define("qx.ui.window.Window",
     */
 
     /**
-     * Closes the current window instance.
-     * Technically calls the {@link qx.ui.core.Widget#hide} method.
+     * Close the current window instance.
+     *
+     * Simply calls the {@link qx.ui.core.Widget#hide} method if the
+     * {@link qx.ui.win.Window#autoDestroy} property is false; otherwise 
+     * removes and destroys the window.
      */
     close : function()
     {
-      if (!this.isVisible()) {
+      if (!this.getAutoDestroy() && !this.isVisible()) {
```

Comment:
Why can't I close a window where *autoDestroy* is set to *true*? Not sure what that check is good foor, can you explain?